mitoinstaller/mitoinstaller/installer_steps/mitosheet_installer_steps.py

```diff
@@ -51,17 +52,17 @@ def remove_mitosheet_3_if_present():
     
 
 def install_step_mitosheet_install_mitosheet():
-    print("This might take a few moments...")
     try:
         install_pip_packages('mitosheet', test_pypi='--test-pypi' in sys.argv)
     except:
         # If the user hits an install error because of permission issues, we ask them if 
         # they want to try a user install
+        # TODO: note this is off for now, as it conflicts with the constant print
+        # messages that we are getting from the install process.
         error_traceback_last_line = get_recent_traceback().strip().split('\n')[-1].strip()
-        if error_traceback_last_line == 'Consider using the `--user` option or check the permissions.':
+        if False or error_traceback_last_line == 'Consider using the `--user` option or check the permissions.':
```

Comment:
Based on the comment above, I think this should be an `and` condition instead of an `or`